src/components/NavBar/NavBar.js

```diff
@@ -0,0 +1,197 @@
+import { useState } from "react";
+import ham from "./ham.png";
+import logotext from "./logotext.png";
+
+export const NavBar = ({ selectTab }) => {
+  const handleToggle = () => setNavbarOpen((prev) => !prev);
+  const [navbarOpen, setNavbarOpen] = useState(false);
+
+  return (
+    <nav className="navBar">
+      <img src={ham} className="icon-ham" onClick={handleToggle}></img>
+
+      <div className={`menuNav ${navbarOpen ? " showMenu" : ""}`}>
+        <div
+          style={{ boxShadow: "rgba(0, 0, 0, 0.1) 0px 1px 10px" }}
+          className=" flex-row justify-space-between "
+        >
+             <a href="https://gorti.netlify.app/">
+          <img src={logotext} className="logo-text pad1rem" /> </a>
+          <span
+            className="button-dismiss"
+            onClick={handleToggle}
+            style={{
+              fontSize: "35px",
+              color: "#023e8a",
+              position: "relative",
+              top: "-5px",
+              padding: "0px 10px 0px 0px",
+              cursor: "pointer"
+            }}
+          >
+            ×{" "}
+          </span>
+        </div>
+        <div
+          style={{
+            fontSize: "0.7rem",
+            fontWeight: "700",
+            opacity: "0.9",
+            marginTop: "1.5rem",
+            marginLeft: "3rem",
+            color: "#023e8a",
+            padding: "0.5rem 0rem"
+          }}
+        >
+          GETTING STARTED
+        </div>
+        <div className="hamMenu-items">
+          <div
+            onClick={() => {
+              selectTab("Installation");
+              handleToggle();
+            }}
+            className="hamMenu-div"
+          >
+            Installation
+          </div>
+        </div>
+
+        <div
+          style={{
+            fontSize: "0.7rem",
+            fontWeight: "700",
+            opacity: "0.9",
+            marginTop: "1.5rem",
+            marginLeft: "3rem",
+            color: "#023e8a",
+            padding: "0.5rem 0rem"
+          }}
+        >
+          COMPONENTS
```

Comment:
Just a suggestion, if you want to display your texts in UI as uppercase, then you can use css property 
`text-transform: uppercase;`.